app/channels/room_channel.rb

```diff
@@ -2,6 +2,12 @@ class RoomChannel < ApplicationCable::Channel
   # calls when a client connects to the server
   def subscribed
     conversation ? stream_for(conversation) : reject
+    conversation&.online_user(current_user)
```

Comment:
No need for safe navigator (`&`) here I think, since you reject the subscription if the conversation was not found

Reply:
I was expecting that behavior too. But if i remove the `&` i get the following error when testing room_channel

```
 NoMethodError:
       undefined method `online_user' for nil:NilClass
```